pymc3/gp/cov.py

```diff
@@ -136,30 +181,45 @@ def __init__(self, input_dim, lengthscales, active_dims=None):
         super(Stationary, self).__init__(input_dim, active_dims)
         self.lengthscales = tt.as_tensor_variable(lengthscales)
 
-    def square_dist(self, X, Z):
+    def square_dist(self, X, Xs):
```

Comment:
A docstring or comment showing how your operations are equivalent to the useful definitions would be useful here.